prime-dotnet-webapi/Services/EmailService.cs

```diff
@@ -390,6 +393,19 @@ public async Task SendEnrolleeAbsenceNotificationEmailAsync(int enrolleeId, Enro
 
         private async Task Send(Email email)
         {
+            var doNotEmail = await _context.DoNotEmail
+                .Where(e => e.Email == string.Join(",", email.To))
+                .Select(e => new {
+                    e.Email,
+                    e.Id
+                })
+                .SingleOrDefaultAsync();
+
+            if (doNotEmail != null) {
+                await _businessEventService.CreateEmailEventAsync(doNotEmail.Id, "Do not email enrollee");
```

Comment:
The first parameter of the CreateEmailEventAsync function is for EnrolleeId. 
![image](https://user-images.githubusercontent.com/108699279/211919441-96e87b08-7e88-435c-bdac-c57f4a540cb3.png)
Since we don't have EnrolleeId in this case, we might just pass 0, or make the datatype nullable int and pass Null. I know that the column EnrolleeId in BusinessEvent table is nullable.

Reply:
Why would the enrolleeId not be added to the table? Maybe I am misunderstanding how the BusinessEvent is utilized but don't we need the ID to know which user to log the event for?

Reply:
Good question. The core function "Send(email)" does not know where calling it. It just sends out email. It could be enrollee sending out email to the vendor to set up new account or Admin sending email to enrollee or the system sending out email to enrollees for renewal. I think using CreateEmailEventAsync() is the correct choice to log the event. However, we don't have an Enrollee ID for every situation. About auditing, the system should have the UserID who triggered the event to store in the CreatedUserId of the BusinessEvent Table. Hope that makes sense.

Reply:
I understand now that makes sense. Thanks for clarifying:)